airbyte-integrations/bases/debezium/src/main/java/io/airbyte/integrations/debezium/internals/DebeziumPropertiesManager.java

```diff
@@ -55,7 +55,8 @@ protected Properties getDebeziumProperties() {
     props.setProperty("max.queue.size", "8192");
 
     props.setProperty("errors.max.retries", "10");
-    props.setProperty("errors.retry.delay.initial.ms", "1000");
+    props.setProperty("errors.retry.delay.initial.ms", "300");
```

Comment:
Cause in our Airbyte code we only wait for 60 seconds for Debezium to return a record. So Debezium needs to hurry and restart in case of an error. If it waits for 10 seconds, there is a possibility by the time it restarts, we would ask it to shutdown anyway